### DISCUSSION TITLE
Rose UM job will not compile

### DISCUSSION THREAD

**Discussion by StevenHardiman** [MetOffice/simulation-systems#145] [UM]
Hi all.  I have a job at vn11.5 which I've run successfully several times before.  But, having copied to a new job, it now won't even compile!  I was wondering if anyone understands the following error please:

[FAIL] file:/home/d05/hadvh/cylc-run/u-dd971/bin=source=fcm:decouputils_tr/bin/@21539: bad or missing value
2024-02-27T15:00:47Z CRITICAL - failed/EXIT

Many thanks!  Steve.

**Comment by richardgilham** [comment]
Have a look at the messaging around changes to the Met Office internal repositories over on Yammer in the Scientific Software Tools community.

**Reply by StevenHardiman** [reply]
Thank you Richard.  OK so these moved last week - that makes sense!  Many thanks!
